ui/components/app/signature-request-original/signature-request-original.container.js

```diff
@@ -31,6 +31,8 @@ function mapStateToProps(state, ownProps) {
   const isLedgerWallet = isAddressLedger(state, from);
   const messagesList = unconfirmedMessagesHashSelector(state);
   const messagesCount = getTotalUnapprovedMessagesCount(state);
+  const currentNetwork =
+    state.metamask.provider.nickname || state.metamask.provider.ticker;
```

Comment:
Does a provider ever not have a nickname?

I think there should always be a name, although for networks that we include within MetaMask, the name likely has t be retrieved by importing the appropriate constants map and using the network chainId. I don't think that the ticker is an appropriate fallback for the network name.

Reply:
The `getNetworkName` method in `network-dropdown.js` provides an example of how this can be done.

Reply:
Testnet networks didn't have a nickname, that's why I used ticker instead.

I updated my PR to use the same function as in network-dropdown.js